### DISCUSSION TITLE
Astroid template Zero vs Astroid template One

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#288] [Q&A] [answered]
Hello everybody,
I'm coming back to Astroid after a short period at Joomlart (T4 Framework).... my experience at Joomlart has not been the best...

Just a simple question about “Zero” and “One” templates, what is exactly the difference between these two templates?
I'm trying to understand what makes them different, but except from some changes in graphic styles I don't see where the difference is? Maybe in the code...? BS4 vs B5?

Thanks!
Lorenzo

**Comment by Chacapamac** [comment]
I @iorbita 
Yes, before  I change to Astroid, I was using Rockettheme for Years, until the change their framework to Gantry. a bad bloated framework that try to reinvent how we supposed to work in Joomla. 
I embark on a year of testing of ALL big Joomla Framework (and Drag & Drop Editors) and finally come across **Joomdev Astroid**. 
**I really see the light!**
I was really disappointed on all these “big” Frameworks (like (T4 Framework) and Astroid even at that time blow them all out of the water for its flexibility, speed, clean minimalist and modern code and on top completely Joomla friendly by using and even enhancing the core Joomla Power and functions.

I use both **Zero** and **One**, they seem to behave the same, I never ask or investigate what was the difference between them...

**@sonvnn can maybe tell us what are the main difference, that will be cool to know. Maybe these differences are important enough to go in the documentation?**

**Good Day! Lorenzo**

**Comment by sonvnn** [comment, marked as answer]
Hi @iorbita @Chacapamac 

Thank you for your support. I glad to answer your question. Zero and One are basic templates of Astroid Framework. They are different in style only. All core functions are same because they were loaded by Astroid Core. All Astroid template support BS 5.3 latest version.

Please let me know if you have any questions.

Thanks & Best Regards,
Sonny

**Reply by Chacapamac** [reply]
Thank You Sonny for the info!

**Good Day everybody!**

**Comment by iorbita** [comment]
Hi,
thank you both, so the only difference are styles and both templates support the latest version of Bootstrap, that's a very good news ;)
I'm so glad to be back to Astroid, I find it so user friendly, I admit I haven't used all frameworks available for Joomla but I tested a lot of them and I find Astroid the most accomplished.
I loved Zengrid framework from Joomlabamboo, but this is another time, another story... Joomlart decided to let it die, anyway without Anthony's magic touch nothing would have been the same...

Have a great day and thank you both for your answers,
Lorenzo
